_announcements/2016-07-11-makerfestival.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Makerfestival"
+dateUntil: 2016-07-11
+---
+
+Come speak with us as this year's [Makerfestival](http://makerfestival.ca)! We'll be hosting a **FREE** satellite event **"Introduction to Networks: From the Internet to tomesh.net"** on June, 25 at 1pm (email [hello@tomesh.net](mailto:hello@tomesh.net) to register or [find out more]()) and have a table at the [festival itself](http://makerfestival.ca) on July 9-10!
```

Comment:
Just added -- made a change to how permalinks are generated (in config), added a slug (on post) and then added the link (in announcement)... hahah did I over-complicate this much?!